src/routes/guides/deployment-options/zerops.mdx

```diff
@@ -0,0 +1,147 @@
+
+services:
+  - hostname: app
+    type: static
+    enableSubdomainAccess: true
+```
+
+##### SSR - Node.js:
+```yaml
+project:
+  name: recipe-solidjs
+
+services:
+  - hostname: app
+    type: nodejs@20
+    enableSubdomainAccess: true
+```
+
+## Add zerops.yml to your repository
+The `zerops.yml` configuration file is used to tell Zerops how to build and run your application, it should be placed to the root of your appplication's repository.
+
+Example for **SSR (Server-Side Rendering)** Apps:
+
+Set up the `zerops.yml` file in the root of your SSR project. Make sure the setup parameter's value is the same as the hostname of the service.
+
+```yaml
+zerops:
+  - setup: app
+    build:
+      base: nodejs@20
+      buildCommands:
+        - pnpm i
+        - pnpm build
+      deployFiles:
+        - .output
+        - node_modules
+        - public
+        - package.json
+    run:
+      base: nodejs@20
+      ports:
+        - port: 3000
+          httpSupport: true
+      start: pnpm start
+```
+
+Example for **SSG (Static Site Generation)** Apps:
+
+Set up the `zerops.yml` file in the root of your SSG project. Make sure the setup parameter's value is the same as the hostname of the service.
+
+```yaml
+zerops:
+  - setup: app
+    build:
+      base: nodejs@20
+      buildCommands:
+        - pnpm i
+        - pnpm build
+      deployFiles:
+        - dist/~
+    run:
+      base: static
+```
+Push the changes to your GitHub/GitLab repository (necessary if you are planning to use GitHub/GitLab).
+
+## Deploying your apps
+
+### Triggering the pipeline automatically by connecting Github/Gitlab repository
+You can push your project by [Triggering the pipeline using Zerops CLI](#triggering-the-pipeline-using-githubgitlab) or by connecting the app service with your [GitHub](https://docs.zerops.io/references/github-integration/) / [GitLab](https://docs.zerops.io/references/gitlab-integration) repository from inside the service detail.
+
+
+
+### Triggering the pipeline manually using Zerops CLI
+
+To download the zCLI binary directly, use [zCLI/releases](https://github.com/zeropsio/zcli/releases) or:
+1. Install the Zerops CLI using Terminal.
+```bash
+npm i -g @zerops/zcli
+```
+2. Open Settings > [Access Token Management](https://app.zerops.io/settings/token-management) in the Zerops app and generate a new access token.
+3. Log in using your access token with the following command:
+```bash
+zcli login <token>
+```
+4. Navigate to the root of your app (where zerops.yml is located) and run the following command in Terminal to trigger the deploy:
+```bash
+zcli push
+```
+
+Feel free to check out the [Zerops Docs](http://docs.zerops.io/) and if you're confused by this guide.
+
+### Community Links
+
+- [Deploying Solid (SSR +SSG) on Zerops | One Click Deploy - Video](https://youtu.be/QLPEqZdpbxA)
+- [Deploying Solid SSG on Zerops](https://youtu.be/G7LfJCfP688)
+- [Deploying Solid SSR on Zerops](https://youtu.be/OaRe--UaWjs)
```

Comment:
videos in docs are always tricky to have because they fell outdated quickly.
Could you please move these videos to Zerops docs so you don't need our help flagging if they require update?
I wish we could help you maintain those, but at the moment we're stretched thin.

Reply:
Yuh, i'll just remove them :)